opentelemetry-appender-tracing/benches/logs.rs

```diff
@@ -10,7 +10,7 @@
     | noop_layer_disabled         | 12 ns       |
     | noop_layer_enabled          | 25 ns       |
     | ot_layer_disabled           | 19 ns       |
-    | ot_layer_enabled            | 371 ns      |
+    | ot_layer_enabled            | 270 ns      |
```

Comment:
❤️ this! Could you update stress tests too?

Reply:
updated stress. The benchmark shows improvement from 391 ns -> 305 ns. Updated that too.